src/assets/scripts/aircraft/AircraftFlightManagementSystem.js

```diff
@@ -988,7 +988,9 @@ export default class AircraftFlightManagementSystem {
             return null;
         }
 
-        return `${this.following.star}.${window.airportController.airport_get().icao}`;
+        const { icao } = window.airportController.airport_get();
```

Comment:
VERY NEAT!

And yes. I thought the `...` operator was what destructuring was with expanding arrays?